AK/JsonParser.cpp

```diff
@@ -26,7 +26,7 @@ String JsonParser::consume_and_unescape_string()
             ch = m_input[peek_index];
             if (ch == '"' || ch == '\\')
                 break;
-            if (ch == '\n')
+            if (ch <= 0x1F)
```

Comment:
Instead of hand-writing those checks for different character types, consider using `AK/CharacterTypes.h`. This makes the code much more expressive, and avoids possible off-by-one errors which can quickly happen (not in this case, your code is correct).
```suggestion
            if (is_ascii_c0_control(ch))
```